### PR TITLE
fix: (iOS) Adds ability to handle DTMF codes in Callkit

### PR DESCRIPTION
**Pull request by fmonsalvo**
This PR adds a handler to manage DTMF codes in Call Kit.

### CODE REVIEW

**Comment by jdegger** [comment]
@fmonsalvo Looking good, the code looks fairly straight forward, I have not tested it yes but will soon. Thanks. I have also requested review from @fabriziomoscon

File: ios/RNTwilioVoice/RNTwilioVoice.m [hunk 1]
```diff
@@ -681,6 +681,14 @@ - (void)provider:(CXProvider *)provider performSetMutedCallAction:(CXSetMutedCal
     }
 }
 
+- (void)provider:(CXProvider *)provider performPlayDTMFCallAction:(CXPlayDTMFCallAction *)action {
+  TVOCall *call = self.activeCalls[action.callUUID.UUIDString];
+  if (call && call.state == TVOCallStateConnected) {
+    RCTLogInfo(@"SendDigits %@", action.digits);
```
Comment (fabriziomoscon) [inline ios/RNTwilioVoice/RNTwilioVoice.m:687]: We haven't used `RCTLogInfo()` so far. I would suggest to stick to `NSLog()`.
If you see advantages that I am not aware of by using `RCTLogInfo()`, please open another PR that changes all the occurrences of `NSLog()`

**Review by fabriziomoscon — CHANGES_REQUESTED** [review]
change the log

**Comment by fmonsalvo** [comment]
@fabriziomoscon just changed the line to use NSLog, please review it.

**Pull request merged**